modules/odc-discovering-and-identifying-operator-backed-bindable-services.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: PROCEDURE
+[id="odc-discovering-and-identifying-operator-backed-bindable-services_{context}"]
+== Discovering and identifying Operator-backed bindable services
+
+As an operator provider or a user who creates backing services, you must know which services are bindable before you create a backing service. The *Developer* perspective helps you discover and identify such bindable services. 
```

Comment:
```suggestion
As a user if you want to create a bindable service, you must know which services are bindable. The *Developer* perspective helps you discover and identify such bindable services. 
```

Reply:
If you want to highlight what are bindable services you can use this - `Bindable services are services that can be easily consumed by applications because they expose their binding data (credentials, connection details, volume mounts, secrets, etc.) in a standard way.`

Reply:
Can you keep the above text as is because that is more technically correct?

Reply:
@divyanshiGupta Could you please confirm now?
![image](https://user-images.githubusercontent.com/89447174/179499996-070535ce-04d1-4fca-9156-b027cd16511c.png)

Thank you!